src/App.js

```diff
@@ -99,7 +117,7 @@ class TodoList extends React.Component {
     return (
       <ul>
         {this.props.items.map(item => (
-          <li key={item.id}><input type='checkbox'/>{item.text}<button onClick={this.props.delete}><img src={trashcan} alt="&#128465;"/></button></li>
+          <li key={item.id}><input type='checkbox'/>{item.text}<button onClick={(e) => this.props.delete(e, item)} aria-label="delete"><img src={trashcan} alt=""/></button></li>
```

Comment:
Looks like I left in my change to pass two props to the delete function, the event and the item. That should give you another hint as to what needs to happen at the function definition of 'delete'.